### ISSUE TITLE
Wrong behaviour when navigating stacked panels

### ISSUE THREAD

**Issue by Lau-San** [hiasr/vim-zellij-navigator#14]
On a layout where I have 3 stacked panels, Pressing C-j while I'm on the topmost panel should switch to the center panel, but it's switching to the bottom panel, and viceversa. This makes the middle panel unreachable unless I go in a roundabout way (leaving the stack and entering it again will work, but yeah, not ideal)

https://github.com/user-attachments/assets/712d05f1-7aa7-4465-9f73-a63cae04bdbb

**Comment by hiasr** [comment]
Apologies for the late response! I believe this is an issue of Zellij itself, do you experience the same issue when not using the plugin?

**Comment by Lau-San** [comment]
> Apologies for the late response! I believe this is an issue of Zellij itself, do you experience the same issue when not using the plugin?

Yup, I can confirm it worked without the plugin, I only started seeing this after installing the it.

However I did upgrade Zellij not too long ago and haven't tested it since. It's late right now and I'm not on my PC, so give me a few hours and I'll check again just in case.

**Comment by Lau-San** [comment]
Just checked, now it works on the latest Zellij version. Not sure what fixed it, but I'll take it!
